tests/test_normalized.cpp

```diff
@@ -0,0 +1,221 @@
+/*****************************************************************************
+ *   Copyright (C) 2004-2013 The PaGMO development team,                     *
+ *   Advanced Concepts Team (ACT), European Space Agency (ESA)               *
+ *   http://apps.sourceforge.net/mediawiki/pagmo                             *
+ *   http://apps.sourceforge.net/mediawiki/pagmo/index.php?title=Developers  *
+ *   http://apps.sourceforge.net/mediawiki/pagmo/index.php?title=Credits     *
+ *   act@esa.int                                                             *
+ *                                                                           *
+ *   This program is free software; you can redistribute it and/or modify    *
+ *   it under the terms of the GNU General Public License as published by    *
+ *   the Free Software Foundation; either version 2 of the License, or       *
+ *   (at your option) any later version.                                     *
+ *                                                                           *
+ *   This program is distributed in the hope that it will be useful,         *
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of          *
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the           *
+ *   GNU General Public License for more details.                            *
+ *                                                                           *
+ *   You should have received a copy of the GNU General Public License       *
+ *   along with this program; if not, write to the                           *
+ *   Free Software Foundation, Inc.,                                         *
+ *   59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.               *
+ *****************************************************************************/
+
+// Test code for the normalized meta-problem
+
+#include <iostream>
+#include <algorithm>
+#include <cmath>
+#include <vector>
+#include <cassert>
+#include "../src/pagmo.h"
+#include "../src/rng.h"
+#include <boost/random/uniform_real.hpp>
+
+using namespace pagmo;
+
+const double EPS = 10e-9;
+
+#define PRINT_VEC(x) do {                             \
+	std::cout << "[ " #x " ] = ";                     \
+	for(unsigned int iii=0; iii<(x).size(); iii++) {  \
+		std::cout<<(x)[iii]<<" ";                     \
+	}                                                 \
+	std::cout<<std::endl; } while(0);
+
+bool is_eq(fitness_vector f1, fitness_vector f2, double eps)
+{
+	if (f1.size() != f2.size()) {
+		return false;
+	}
+	for (size_t i = 0; i < f1.size(); i++) {
+		if (fabs(f1[i] - f2[i]) > eps) {
+			return false;
+		}
+	}
+	return true;
+}
+
+// Test following invariant:
+// normalized(prob).objfun(dv) is equal to
+//   prob.objfun(normalized(prob).denormalize(dv))
+int test_normalized_invariant(
```

Comment:
Merge the lines here 63 and 64.

Method commenting style is either: https://github.com/esa/pagmo/blob/master/src/problem/normalized.cpp#L35-L41
or triple slash like https://github.com/esa/pagmo/blob/master/src/problem/normalized.cpp#L58 
Latter is fine for unit testing.